index.html

```diff
@@ -6,6 +6,6 @@
     <title>Document</title>
 </head>
 <body>
-    Welcome To Potionfy! You are Awesome 
+    Welcome To Potionfy! Enjoy your Stay 
```

Comment:
Can you remove "Welcome To Potionfy! Enjoy your Stay" to "Welcome To Potionfy!" string red